Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/FileShareServiceTest.cs

```diff
@@ -48,22 +50,50 @@ public void Setup()
         public void WhenFileShareServiceCallsFssBatchSearchAsync_ThenReturnsBatchSearchResponse()
         {
             string searchText = "";
-            string accessToken = "";
-
             IResult<BatchSearchResponse> expected = new Result<BatchSearchResponse>();
-           
+
             A.CallTo(() => _fileShareApiClient.Search("", 100, 0, CancellationToken.None)).Returns(expected);
-            Task<IResult<BatchSearchResponse>> result = _fileShareService.FssBatchSearchAsync(searchText, accessToken , CorrelationId);
-            Assert.IsInstanceOf<Task<IResult<BatchSearchResponse>>>(result);
+            Task<IResult<BatchSearchResponse>> result = _fileShareService.FssBatchSearchAsync(searchText, FakeAccessToken, CorrelationId);
+            Assert.IsInstanceOf<Task<IResult<BatchSearchResponse>>>(result);            
         }
 
         [Test]
-        public async Task WhenFileShareServiceCallsFssBatchSearchAsyncWithInvalidData_ThenReturnsException()
+        public void WhenFileShareServiceCallsFssBatchSearchAsyncWithInvalidData_ThenReturnsException()
         {
             _fileShareServiceConfig.Value.PageSize = -100;
+            _fileShareServiceConfig.Value.BaseUrl = "https://www.test.com/";
             A.CallTo(() => _fileShareApiClient.Search(A<string>.Ignored, A<int>.Ignored, A<int>.Ignored, A<CancellationToken>.Ignored));
-            IResult<BatchSearchResponse> result = await _fileShareService.FssBatchSearchAsync("", "", CorrelationId);
-            Assert.That(result.IsSuccess,Is.False);
+
+            Assert.ThrowsAsync(Is.TypeOf<ArgumentException>()
+                   .And.Message.EqualTo("Page size must be greater than zero. (Parameter 'pageSize')")
+                    , async delegate { await _fileShareService.FssBatchSearchAsync("", "", CorrelationId); });
+        }
+
+        [Test]
+        public void WhenFileShareServiceCallsFssSearchAttributeAsync_ThenReturnsBatchSearchResponse()
+        {
+            int ExpectedStatusCode = 200;
+            bool ExpectedStatus = true;
+            _fileShareServiceConfig.Value.BaseUrl = "https://www.test.com/";
+            IResult<BatchAttributesSearchResponse> expectedResponse = new Result<BatchAttributesSearchResponse>();
+            A.CallTo(() => _fileShareApiClient.BatchAttributeSearch("", CancellationToken.None)).Returns(expectedResponse);
+
+            Task<IResult<BatchAttributesSearchResponse>> result = _fileShareService.FssSearchAttributeAsync(FakeAccessToken, CorrelationId);
+
+            Assert.IsInstanceOf<Task<IResult<BatchAttributesSearchResponse>>>(result);
+            Assert.AreEqual(result.Result.StatusCode, ExpectedStatusCode);
+            Assert.AreEqual(result.Result.IsSuccess, ExpectedStatus);
```

Comment:
Signature of `AreEqual` is:
```
public static void AreEqual(object? expected, object? actual)
```

so:
```suggestion
            Assert.AreEqual(expectedStatusCode, result.Result.StatusCode);
            Assert.AreEqual(expectedStatus, result.Result.IsSuccess);
```

Reply:
corrected